app/index.html

```diff
@@ -8,6 +8,10 @@
         body {
             -webkit-app-region: drag
         }
+        button,
+        input[type] {
```

Comment:
Why `input[type]` ?
Why not just `input`?

Reply:
It matches only inputs which have a type attribute. TBH I'm not sure that's enough, but it seems to work well in practice 😄

Reply:
But something is not working if we use just input?

Reply:
I don’t know, we can change it.